vmdb/app/models/ems_refresh/save_inventory_orchestration_stacks.rb

```diff
@@ -15,11 +15,18 @@ def save_orchestration_templates_inventory(_ems, hashes, _target = nil)
       hashes.zip(templates).each { |hash, template| hash[:id] = template.id }
     end
 
+    def all_orchestration_stacks(ems)
+      if ems.respond_to?(:orchestration_stacks_with_nested)
+        ems.orchestration_stacks_with_nested(true)
+      else
+        ems.orchestration_stacks(true)
+      end
+    end
+
```

Comment:
Yes. I passed the test.
Don't make the change yet. I will need to discuss with the UI team how to present nested stacks. Currently there is no tree view for nested stacks, all are showed as flat.